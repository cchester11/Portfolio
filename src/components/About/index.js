import React from 'react';
import solarsytem from '../../assets/cover/solarsystem.jpeg'

function About () {
  return (
    <section>
      <h1 id="about">Charles Chester</h1>
      <img src={solarsytem} className="my-2" style={{width: "100%"}} alt="cover" />
    </section>
  )
}

export default About;