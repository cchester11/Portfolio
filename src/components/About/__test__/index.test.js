import React from 'react';
import { cleanup, render } from '@testing-library/react';
import '@testing-library/jest-dom/extend-expect';
import About from '..';

afterEach(cleanup);

describe("About component", () => {
  //make sure 'it' renders correctly using
  it('renders', () => {
    render(<About />)
  })

  //make sure the rendered component matches snapshot 
  it("matches snapshot DOM node structure", () => {
    const {asFragment} = render(<About />)
    expect(asFragment()).toMatchSnapshot();
  })
})