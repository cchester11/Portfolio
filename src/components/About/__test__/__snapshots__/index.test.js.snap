// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`About component matches snapshot DOM node structure 1`] = `
<DocumentFragment>
  <section>
    <h1
      id="about"
    >
      Charles Chester
    </h1>
    <img
      alt="cover"
      class="my-2"
      src="solarsystem.jpeg"
      style="width: 100%;"
    />
  </section>
</DocumentFragment>
`;
